opencog/atoms/execution/ExecutionOutputLink.cc

```diff
@@ -195,6 +196,49 @@ Handle ExecutionOutputLink::do_execute(AtomSpace* as,
 #endif /* HAVE_CYTHON */
 	}
 
+	if (0 == schema.compare(0,4,"lib:", 4))
+	{
+		// Be friendly, and strip leading white-space, if any.
+		size_t pos = 4;
+		while (' ' == schema[pos]) pos++;
+
+        //Get the name of the Library and Function
+        //They should be sperated by a .
+        std::size_t dotpos = schema.find("\\");
+        if (dotpos == std::string::npos)
+        {
+            throw RuntimeException(TRACE_INFO,
+                "LibName and FunctionName not seperated by a '\\'");
+        }
+        std::string libName  = schema.substr(pos,dotpos-pos);
+        std::string funcName = schema.substr(dotpos+1);
+
+        void *libHandle;
+        void * tmp;
+        UUID (*func)(AtomSpace* ,UUID);
```

Comment:
Well that wasn't my decision, it's how the Haskell Bindings are written but i could rewrite them to use handles at some point.